lib/WebService/Stripe.pm

```diff
@@ -173,6 +173,12 @@ method cancel_transfer(Str $id, :$headers) {
     return $self->post("/v1/transfers/$id/cancel", undef, headers => $headers);
 }
 
+method create_reversal($xfer_id, HashRef $data = {}, HashRef $opts = {}) {
```

Comment:
I guess I'm confused on the inconsistency with the other transfer methods (or other methods in general). It appears we pass headers => { 'stripe_account' => foo } for them. Is there some magic somewhere that turns that into 'Stripe-Account:'...   (I like what you have here better, but just wondering)